code/game/objects/items/ego_weapons/he.dm

```diff
@@ -1345,4 +1345,4 @@
 	block_sound_volume = 200
 	attribute_requirements = list(
 							FORTITUDE_ATTRIBUTE = 40
-							)
+							)
```

Comment:
```suggestion
							)
							
```
I don't know why this was removed, but I've re-added it. This will fix your linters issue.